readthedocs/projects/tasks.py

```diff
@@ -1324,17 +1324,13 @@ def _manage_imported_files(version, path, commit):
     )
     # Keep the objects into memory to send it to signal
     instance_list = list(delete_queryset)
```

Comment:
Doing `list()` doesn't make to hit the db already?

Reply:
Yea, we can probably remove this if we want, since the objects should exist now.